theme/less/main.less

```diff
@@ -4,20 +4,37 @@
 body
 {
   margin: 0px;
-  font-family: sans-serif;
+  font-family: "PT Sans", "HelveticaNeue", "Helvetica Neue", Helvetica, Arial, sans-serif;
 }
 
 header
 {
-  background-color: @primary;
-  padding: 30px;
+  padding: 30px; 
+  margin: 0px auto;
+  max-width: 1000px;
 }
 
-header > div
+header > p
 {
-  margin: 0px auto;
-  max-width: 1000px;
-  color: @reverse;
+  color: #7d7d7d;
+  font-size: 24px;
+  letter-spacing: -1px;
+}
+
+header h1, header h1 a
```

Comment:
Suggestion: We do apply duplicate styles here, while in some cases it would be good idea, here we are missing some optimization opportunities. we can either use LESS here, or move `header h1 a ` selector from here:
```
header h1 a
{
  color: #262626;
  text-decoration: none;
}
```

After we can also notice that there are some common styles in whole header anchor elements, if needed and you want we can also improve further and have:
```
header a
{
  text-decoration: none;
}
```

That way we don't need to specify that rule anywhere else in the header and simplify our selectors, just a small suggestion.

Reply:
Good option! Done.